### PR TITLE
Fixed some syntax issues in Twig Reference

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | - |

### CODE REVIEW

File: reference/twig_reference.rst [hunk 1]
```diff
@@ -95,7 +95,7 @@ controller
     **type**: ``array`` **default**: ``[]``
 
 Returns an instance of ``ControllerReference`` to be used with functions like
-:ref:`render() <reference-twig-function-render>` and `render_esi() <render_esi>`.
+:ref:`render() <reference-twig-function-render>` and :ref:`render_esi() <render_esi>`.
```
Comment (xabbuh) [inline reference/twig_reference.rst]: You will have to add a label before the "render_esi" headline like this:

```rst
.. _reference-twig-function-render-esi:
```

and then change the referenced label name here accordingly.

File: reference/twig_reference.rst [hunk 2]
```diff
@@ -10,8 +10,8 @@ Twig is the default template engine for Symfony. By itself, it already contains
 a lot of built-in functions, filters, tags and tests (learn more about them
 from the `Twig Reference`_).
 
-Symfony adds more custom extensions on top of Twig to integrate some components
-into the Twig templates. You can find more information about the custom
+Symfony adds some custom extensions on top of Twig to integrate some components
```
Comment (timglabisch) [inline reference/twig_reference.rst]: why not just Symfony adds custom extensions .... to avoid some 2 times.

**Comment by javiereguiluz** [comment]
@xabbuh I've made the required changes. I hope everything is OK now.

**Comment by WouterJ** [comment]
Thank you Javier! I'm shocked with the amount of typos I made :)

**Pull request closed without merging**